src/data/api.ts

```diff
@@ -322,6 +322,20 @@ export const GetUserMetadata = async (walletAddr: string) => {
   return tzktData
 }
 
+/**
+ * Get User DAO token balance
+ */
+export async function getDaoTokenBalance(walletAddr: string) {
+  const parameters = {
+    'token.contract': DAO_TOKEN_CONTRACT,
+    'token.tokenId': '0',
+    'account': walletAddr,
+    "select": 'balance'}
+  const response = await axios.get(`https://api.tzkt.io/v1/tokens/balances`, { params: parameters })
+    .catch(error => console.log('Error while querying the account token balance:', error))
+
+  return response?.data[0]? parseInt(response.data[0]) / 1e6 : 0
+}
 
```

Comment:
We probably can also use teztok here? I guess it doesn't matter that much

Reply:
Nope it's actually not indexed and IIRC @xat mentionned it not being very adequate to store there so this is totally fine